intelmq/bots/collectors/http/collector_http.py

```diff
@@ -29,11 +29,25 @@ def process(self):
         self.logger.info("Downloading report from %s" %
                          self.parameters.http_url)
 
-        resp = requests.get(url=self.parameters.http_url, auth=self.auth,
-                            proxies=self.proxy, headers=self.http_header,
-                            verify=self.http_verify_cert,
-                            cert=self.ssl_client_cert,
-                            timeout=self.http_timeout)
+        timeoutretries = 0
+        resp = None
+
+        while timeoutretries < self.http_timeout_max_tries and resp is None:
```

Comment:
@wagner-certat can you validate this with me:

Is the Error Handling features documented [here](https://github.com/certtools/intelmq/blob/master/docs/User-Guide.md#error-handling) capable to do the same thing as this pull request want to do? I'm not 100% sure but it seems the same.